docs/mycoind/setup.rst

```diff
@@ -10,30 +10,34 @@ for both go and node.
 **WARNING**
 
 This is only tested under Linux and OSX.
-It will most likely not work under Windows, even with Cygwin.
-If you use windows, please make a PR with any adjustments so
-this tutorial works for Windows as well.
+If you want to run under windows, the only supported *development* environment
+is using WSL (Windows Subsytem for Linux) under Windows 10.
+Follow `these directions <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`__
+to setup ubuntu in WSL, then try the rest in your Ubuntu shell
```

Comment:
I think naming capitalization should be consistent. I prefer `Windows` and `Ubuntu` (not `windows` and `ubuntu`) :nail_care: